src/Nethermind/Nethermind.Merge.Plugin/Handlers/PayloadService.cs

```diff
@@ -68,7 +71,7 @@ public class PayloadService : IPayloadService
         public async Task<byte[]> StartPreparingPayload(BlockHeader parentHeader, PayloadAttributes payloadAttributes)
         {
             byte[] payloadId = ComputeNextPayloadId(parentHeader, payloadAttributes);
-            payloadAttributes.SuggestedFeeRecipient = payloadAttributes.SuggestedFeeRecipient == Address.Zero ? _sealer.Address : payloadAttributes.SuggestedFeeRecipient;
+            payloadAttributes.SuggestedFeeRecipient = _mergeConfig.BlockAuthorAccount != "" ? new Address(_mergeConfig.BlockAuthorAccount) : payloadAttributes.SuggestedFeeRecipient;
```

Comment:
take a look in MergeSealEngine.cs, we need address from this class ;) we can talk about it  :)